jax/experimental/jax2tf/jax2tf.py

```diff
@@ -876,7 +876,18 @@ def _population_count(x):
 tf_impl[lax.bessel_i1e_p] = tf.math.bessel_i1e
 
 tf_impl[lax.complex_p] = tf.complex
-tf_impl[lax.conj_p] = tf.math.conj
+
+def _conj(x, **kwargs):
+  # The only dtypes that are allowed are: float32, float64, complex64, and
+  # complex128.
+  if x.dtype == tf.float32:
```

Comment:
was the problem here that `tf.math.conj` returns a float32 if the input is a float32, while HLO returns a complex64?

Reply:
Yes!